src/ILLink.Shared/ILLink.Shared.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is it really worth adding complexities with managing a shared library for 4 lines of c# code that could be easily shared differently?

Reply:
They share the resource file, and while at this moment only contains IL2026, the amount of possible shared information will increase. Only from other PR that I have open it's other 4 diagnostics that could be shared and support for DynamicallyAccessMembers in the analyzer would make sharing the diagnostics even more important.

Reply:
Was there any problem sharing the resource file as a build input?

Reply:
Not sure what you mean by that -- you mean sharing by including files into both projects?

Reply:
Changed to compile the shared files in separately. It's a little bit tricky when dealing with shared resources, but presumably we shouldn't have to touch that very often.